system/Database/Forge.php

```diff
@@ -456,6 +444,22 @@ public function dropKey(string $table, string $keyName)
         return $this->db->query($sql);
     }
 
```

Comment:
Added new method dropPrimaryKey(). The existing methods didn't allow for dropping primary.